web/src/components/PoiDetails.tsx

```diff
@@ -132,22 +125,13 @@ const PoiDetails = ({ poi, distance, toolbar }: PoiDetailsProps): ReactElement =
   const { viewportSmall } = useWindowDimensions()
   const theme = useTheme()
   const { t } = useTranslation('pois')
-  const {
-    content,
-    location,
-    website,
-    phoneNumber,
-    email,
-    isCurrentlyOpen,
-    openingHours,
-    temporarilyClosed,
-    category,
-    appointmentUrl,
-  } = poi
+  const { content, location, contacts, isCurrentlyOpen, openingHours, temporarilyClosed, category, appointmentUrl } =
+    poi
 
   const thumbnail = poi.thumbnail ?? PoiThumbnailPlaceholderLarge
   const isAndroid = /Android/i.test(navigator.userAgent)
   const externalMapsLink = getExternalMapsLink(location, isAndroid ? 'android' : 'web')
+  const appointmentOverlayLink = appointmentUrl ?? poi.contacts.find(contact => contact.website !== null)?.website ?? ''
```

Comment:
:wrench: See comment for native.
```suggestion
  const appointmentUrl = poi.appointmentUrl ?? poi.contacts.find(contact => contact.website !== null)?.website ?? null
```